src/AdyenNativeModules.tsx

```diff
@@ -200,14 +233,12 @@ export function getNativeComponent(
     case 'drop-in':
     case 'adyendropin':
       return {
-        nativeComponent: new AdyenNativeComponentWrapper(AdyenDropIn, true, [
-          Event.onComplete,
-        ]),
+        nativeComponent: new AdyenNativeComponentWrapper(AdyenDropIn),
         paymentMethod: undefined,
       };
     case 'applepay':
       return {
-        nativeComponent: new AdyenNativeComponentWrapper(AdyenApplePay, false),
+        nativeComponent: new AdyenNativeComponentWrapper(AdyenApplePay, true, false),
```

Comment:
Hmm maybe we can refactor the `AdyenNativeComponentWrapper` constructor to something like this for the readability reason, because we can easily forget the order of `canHandlePayment` and `canHandleAction` in the parameter list.. but it's optional, up to you :)

```typescript
interface AdyenNativeComponentWrapperProps {
  nativeModule: NativeModule,
  canHandlePayment?: boolean,
  canHandleAction?: boolean,
  events?: string[]
}

constructor({
nativeModule,
canHandlePayment=true,
canHandleAction=true,
events}:AdyenNativeComponentWrapperProps) {
// the rest
}

```

Reply:
Good one, thanks!
Fixed in [9b991ef](https://github.com/Adyen/adyen-react-native/pull/331/commits/9b991ef9d4ad199e254be132b1cd4e006a2ec8f9)